addon-test-support/@ember/test-helpers/-internal/build-registry.ts

```diff
@@ -58,9 +58,11 @@ const Owner = EmberObject.extend(RegistryProxyMixin, ContainerProxyMixin, {
    * @see {@link https://github.com/emberjs/ember.js/blob/v4.5.0-alpha.5/packages/%40ember/engine/instance.ts#L152-L167}
    */
   unregister(fullName: string) {
+    // @ts-expect-error
```

Comment:
These types aren't public and there is a bunch of `EmberObject` and `as any` in this file, so I figured leaving `@ts-expect-error` was OK...

Reply:
That's the best move, yep. We can flag it up for figuring out a non-private-API way of tackling in the future!